rust/noosphere-ns/src/validator.rs

```diff
@@ -22,17 +22,31 @@ where
     }
 }
 
+impl<S> Clone for Validator<S>
```

Comment:
Should this also implement Display like the AllowAllValidator?

Reply:
Ah, `Display` is vestigial, I can remove that implementation